index.html

```diff
@@ -38,6 +39,131 @@ <h1 class="primary-text">I’m Richmond Glad to see you!</h1>
+            </p>
+            <ul>
+              <li>html</li>
+              <li>css</li>
+              <li>Javascript</li>
+            </ul>
+            <div class="button1"><a href="#">See Project</a></div>
+          </div>
+          <div class="card">
+            <div class="img"><img src="./logos/t5.png" alt="work example" /></div>
+            <p class="card-title">Multi-Post Stories</p>
+            <div class="tag">
+              <ul>
+                <li>Canopy</li>
+                <li>Backend-Dev</li>
+                <li> 2015</li>
+            </ul>
+             </div>
+            <p class="card-text">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul>
+              <li>html</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+            <div class="button1"><a href="#">See Project</a></div>
+          </div>
+          <div class="card">
+            <div class="img"><img src="./logos/t3.png" alt="work example" /></div>
+            <p class="card-title">Tonic</p>
+            <div class="tag">
+              <ul>
+                <li>Canopy</li>
+                <li>Backend-Dev</li>
+                <li> 2015</li>
+            </ul>
+             </div>
+            <p class="card-text">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul>
+              <li>html</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+            <div class="button1"><a href="#">See Project</a></div>
+          </div>
+          <div class="card">
+            <div class="img"><img src="./logos/t4.png" alt="work example" /></div>
+            <p class="card-title">Multi-Post Stories</p>
+            <div class="tag">
+              <ul>
+                <li>Canopy</li>
+                <li>Backend-Dev</li>
+                <li> 2015</li>
+            </ul>
+             </div>
+            <p class="card-text">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul>
+              <li>html</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+            <div class="button1"><a href="#">See Project</a></div>
+          </div>
+        </div>
+      </section>
+      <section class="container2">
+        <p class="primary-text">About Me</p>
+        <p class="content">
+            I’m a software developer! I can help you build a product , 
+            feature or website Look through some of my work and experience! 
+            If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.
+        </p>
+        <p class="Connect">LET'S CONNECT</p>
+        <ul class="social-media">
```

Comment:
- [ ] "LET'S CONNECT" should use a header tag `hx`(`<h2>`, `<h3>` etc) instead of `p` tag.